sys-process/rtirq/rtirq-20180209.ebuild

```diff
@@ -0,0 +1,42 @@
+# Copyright 1999-2018 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="7"
+
+inherit systemd
+
+DESCRIPTION="Modify realtime scheduling policy and priority of IRQ handlers"
+HOMEPAGE="http://www.rncbc.org/archive/#rtirq"
+
+SRC_URI="http://www.rncbc.org/archive/${P}.tar.gz"
+LICENSE="GPL-2"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+
+src_prepare() {
+	# Correct config file path.
+	sed -i -e "s:^\(RTIRQ_CONFIG\=\)\(.*\):\1/etc/conf.d/rtirq:" ${PN}.sh || die
+	sed -i -e "s:/etc/sysconfig/rtirq:/etc/conf.d/rtirq:" ${PN}.conf || die
+
+	default
+}
+
+src_install(){
+	dosbin ${PN}.sh
+	doinitd "${FILESDIR}"/${PN}
+	newconfd ${PN}.conf ${PN}
+	systemd_dounit ${PN}.service
+}
+
+pkg_postinst(){
+	if [[ "$(rc-config list default | grep rtirq)" = "" ]] ; then
+		elog "You probably want to add rtirq to the default runlevel, i.e."
+		elog "  rc-update add rtirq default"
+	fi
+	if [[ "$(uname -r | grep rt)" = "" ]] ; then
```

Comment:
[[ $(uname -r) == *rt* ]]

There's no need to call `grep` when bash can test it for you.

Also, can't you detect it via some kernel .config option?